docs/reference/rest-api.md

```diff
@@ -107,12 +107,12 @@ The endpoint will return 10 million values if 10 million documents match the que
 #### Get parameters
 
 | Variable            | Type       | Description                                                                                                      | Default value                                      |
-| ----------          | ------     | -------------                                                                                                    | ---------------                                    |
+|---------------------|------------|------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
```

Comment:
formatting only

Reply:
This is pointless. Formatting tables with the minimal markdown necessary is easier to write and read.

```
Variable | Type | Description | Default value
---|---|---|---
foo | bar | baz | qux
```

Variable | Type | Description | Default value
---|---|---|---
foo | bar | baz | qux

Reply:
Not completely pointless but totally useless when tables are big.
We can have a guideline on how to format markdown tables, I was just being consistent throughout the document.

Reply:
I bet you don't how I'm writing the docs on markdown table....

Reply:
zoom out 3 times so that the table is beautiful.... I will surely have problem with my eyes if I'm writing too many markdown tables